test/hotspot/jtreg/ProblemList-Virtual.txt

```diff
@@ -39,15 +39,6 @@ vmTestbase/nsk/jvmti/CompiledMethodUnload/compmethunload001/TestDescription.java
 
 vmTestbase/nsk/jvmti/GetCurrentThreadCpuTime/curthrcputime001/TestDescription.java 8300708 generic-all
 vmTestbase/nsk/jvmti/GetThreadCpuTime/thrcputime001/TestDescription.java 8300708 generic-all
```

Comment:
You still have these two unfixed tests in the problem list using the CR for this PR. Either they will need a new CR, or you will need to use a different CR for this PR.

Reply:
Thanks. I've created a new bug and replaced the bug number in these entries:
  [8348844](https://bugs.openjdk.org/browse/JDK-8348844): Remove remaining JVMTI tests from ProblemList-Virtual, use require instead